Sources/StreamChatUI/ChatMessageList/RETHINK/MessageContentView.swift

```diff
@@ -412,7 +419,7 @@ private extension MessageContentView {
             textView?.isScrollEnabled = false
             textView?.backgroundColor = .clear
             textView?.adjustsFontForContentSizeCategory = true
-            textView?.textContainerInset = .zero
+            textView?.textContainerInset = .init(top: 0, left: 8, bottom: 0, right: 8)
```

Comment:
@VojtaStavik  I was thinking about extending `ContainerView` so it's possible to provide custom layout margins for each arranged subview but decided to discuss `ContainerView` with you first and keep it simple in the PR

Reply:
@evsaev sure, let's talk about the details during auditing 👍